### PR TITLE
riak 2.9.0p1

### PR DESCRIPTION
**Pull request by chenrui333**
relates to https://github.com/Homebrew/homebrew-core/pull/40075

### CODE REVIEW

**Comment by chenrui333** [comment]
```
17:42:49 following command and follow the instructions in your editor to edit
17:42:49 your configuration file:
17:42:49 
17:42:49     git config --global --edit
17:42:49 
17:42:49 After doing this, you may fix the identity used for this commit with:
17:42:49 
17:42:49     git commit --amend --reset-author
17:42:49 
17:42:49  2 files changed, 5 insertions(+), 1 deletion(-)
17:42:49 Error: An exception occurred within a child process:
17:42:49   Utils::InreplaceError: inreplace failed
17:42:49 rebar.config.lock:
17:42:49   expected replacement of "f6ed834cd8799623ec00faaedc9ef2a55876d5d8" with "f8f763c503171602e218def94b13ea72dabb5599"
17:42:49 
```

File: Formula/riak.rb [hunk 1]
```diff
@@ -18,12 +18,12 @@ class Riak < Formula
   # rebar tries to fetch fuse using git over ssh
   resource "fuse" do
     url "https://github.com/jlouis/fuse.git",
-        :revision => "21c6e52ced3af294f2fe636039106068da12eeeb"
+      :tag      => "2.1.0"
```
Comment (zbeekman) [inline Formula/riak.rb]: Can we use the `:revision` this tag points to? People sometimes overwrite tags.

File: Formula/riak.rb [hunk 2]
```diff
@@ -44,7 +44,7 @@ class Riak < Formula
 
   resource "yokozuna" do
     url "https://github.com/basho/yokozuna.git",
-        :revision => "b53d999529626301c36fa3efa22b2b0165217556"
+        :tag      => "riak_kv-2.9.0"
```
Comment (zbeekman) [inline Formula/riak.rb]: Again, can we use the `:revision` that this tag points to? Also, add a comment with the tag to clarify.

**Review by zbeekman — CHANGES_REQUESTED** [review]
Some tweaks for resources... not sure how to address the main problem though.

**Comment by chenrui333** [comment]
No idea about how to fix it, close the PR for now.

**Pull request closed without merging**